### PR TITLE
Restore some state proto documentation

### PR DESCRIPTION
**Pull request by fejta**

### CODE REVIEW

File: pb/state/state.proto [hunk 1]
```diff
@@ -90,20 +90,16 @@ message TestMetadata {
...
 
   // Name associated with the column (such as the run/invocation ID).No two
   // columns should have the same build_id and name. The name field allows the
   // display of multiple columns with the same build_id.
   string name = 2;
 
-  // Timestamp for the column, typically the earliest timestamp of any test
-  // result in the column.
-  double started = 3;
+  double started = 3; // Milliseconds since start of epoc (python time.time() * 1000)
```
Comment (michelle192837) [inline pb/state/state.proto:100]: nit: epoch?

Comment (fejta) [reply pb/state/state.proto:100]: Done

**Pull request merged**